### ISSUE TITLE
Missing implementation of one ConstraintMatrix::add_entries_local_to_global

### ISSUE THREAD

**Issue by luca-heltai** [dealii/dealii#6433]
For the case of standard matrices, the function `ConstraintMatrix::add_entries_local_to_global` that takes different row and column indices, **and** a non-trivial `dof_mask`, is not implemented.

The implementation is there only for block matrices.

**Comment by luca-heltai** [comment]
Related to #6427

**Comment by kronbichler** [comment]
I guess we won't be able to do it now, even though it does not appear like a lot of work.

**Comment by kronbichler** [comment]
Anyone interested in this feature - if not, we should move this to the 9.3 milestone.

**Comment by luca-heltai** [comment]
I agree. I think I'm the only one who noticed this. I have left marks in the codes where this is used, that can be updated when the feature is implemented.

**Comment by drwells** [comment]
This is not a high priority item so I'm removing it from the milestone.
